script.js

```diff
@@ -19,4 +19,110 @@ document.addEventListener('DOMContentLoaded', () => {
+    worksection.innerHTML = container01 + container02 + container03 + container04;
+  }
+  document.querySelector('.close').addEventListener('click', () => {
+    document.querySelector('.popup').classList.remove('popup');
+    document.querySelector('.popup').classList.add('popup-inactive');
+  });
+  document.querySelector('.project-1').addEventListener('click', () => {
+    document.querySelector('.popup1').classList.remove('popup-inactive');
+    document.querySelector('.popup1').classList.add('popup1');
+  });
+  document.querySelector('.project-2').addEventListener('click', () => {
+    document.querySelector('.popup2').classList.remove('popup-inactive');
+    document.querySelector('.popup2').classList.add('popup2');
+  });
+  document.querySelector('.project-3').addEventListener('click', () => {
+    document.querySelector('.popup3').classList.remove('popup-inactive');
+    document.querySelector('.popup3').classList.add('popup3');
+  });
+  document.querySelector('.project-4').addEventListener('click', () => {
+    document.querySelector('.popup4').classList.remove('popup-inactive');
+    document.querySelector('.popup4').classList.add('popup4');
+  });
+  const form = document.querySelector('#form');
+  form.addEventListener('submit', (event) => {
+    const emailInput = document.querySelector('#email').value;
+    const SubmitEmail = emailInput.toLowerCase();
+    if ((emailInput !== SubmitEmail)) {
+      document.querySelector('.failed').classList.add('failed');
+      document.querySelector('.failed').classList.remove('validate-email-disabled');
+      event.preventDefault();
+    } else {
+      document.querySelector('.failed').classList.add('validate-email-disabled');
+      document.querySelector('.failed').classList.remove('failed');
+      form.onSubmit();
```

Comment:
- [Optional] Great Job for showing the validation message, I think it would be better if you styled with bold and red color for better user accessibility 👍